### DISCUSSION TITLE
Construction Timelapse - Troubleshooting

### DISCUSSION THREAD

**Discussion by chooshemataha** [gopro/labs#917] [Q&A]
I have a couple of Hero12 cameras I'm attempting to do a home construction timelapse with. I've been attempting to use "Very Long Time-lapses with Larger Intervals," but the camera battery dies during the night. During the day, it seems to function as expected. For background, these are multi-day timelapses, and during the day, the battery life hardly moves, but once it becomes dark, they die quickly. Any ideas what's going on?

I've also tried experimenting with "Daily Long Time-lapses" this morning, and it seems to not be initiating. The camera stays on with the words, "Pause for 1 secs button to cancel" keeps flashing on the screen, and I see "labs processing" at the top of the screen. At the beginning, after taking a pic of the QR code and seeing the acceptance of it, I then see a QR code on the screed with what appears to be a red equal sign inside of it. It's took quick to really see though. Any thoughts on this?

**Comment by CHEETHAM1** [comment]
Is the gopro turning off between photos?
And are you using the night lapse setting for both day and night?

**Comment by chooshemataha** [comment]
Yes, throughout the day, the gopro is turning off between photos, and the battery is staying above 97%, even though it's on all day.

I am not using the nightlapse setting. In fact, I'm not using any timelapse settings. I'm just taking the image of the QR code, which initiaties the timelapse. I am changing the camera mode to super and linear before I start. Should I also be setting the timelapse mode?

**Comment by CHEETHAM1** [comment]
You're doing better than me already as I only started today:-)
When I scan my code the camera starts doing whatever was on the gopro screen and not what the qrcode was programmed for.
How did you get it to turn off between shots? That doesnt seem to be an option in the settings.

**Comment by chooshemataha** [comment]
It's not an option in the settings, but it is a feature of the "Very Long Time-Lapses With Larger Intervals" feature. From what I can tell though, once you take a picture of the QR code, don't touch the buttons again. I don't know why, but the battery never lasted if I hit a button to validate it was working.
